content/docs/rfcs/51/README.md

```diff
@@ -121,13 +121,37 @@ For this reason, the current specification only supports a single shard cluster
 In future versions, we will add further (hierarchical) discovery methods.
 We will update [31/WAKU2-ENR](https://rfc.vac.dev/spec/31/) accordingly, once this RFC moves forward.
 
-The representation in the ENR is specified as follows.
+This document specifies two ways of indicating shard cluster participation.
+The index list SHOULD be used for nodes that participante in fewer than 64 shards,
```

Comment:
Perhaps something about nodes MUST NOT use both an index list and bit vector representation?

Reply:
And an ENR SHOULD be considered invalid if it contains both `rs` and `rsv` fields and SHOULD be discarded?